content/blog/2018-04-26-rectangling-onboarding.md

```diff
@@ -0,0 +1,498 @@
+---
+slug: rectangling-onboarding
+title: Rectangling onboarding
+authors:
+  - name: Maëlle Salmon
+    url: http://www.masalmon.eu/
+date: 2018-04-26
+preface: "This blog post is the first of a 3-post series about a data-driven overview of rOpenSci onboarding. Read the intro to the series [here](https://ropensci.org/blog/2018/04/26/satrday-ct-serie/)."
```

Comment:
preface is repeated. delete one

![screen shot 2018-04-25 at 11 01 46 pm](https://user-images.githubusercontent.com/11927811/39288435-cb9094be-48dc-11e8-80f6-fe68b8bbeb53.png)